develop/application-devs/smart-contracts/smart-contracts.md

```diff
@@ -0,0 +1,165 @@
+
+1. **Wasm**. The contracts pallet uses WebAssembly as its compilation target. Any language that
+   compiles to Wasm can potentially be used to write smart contracts. Nevertheless, it is better to
+   have a dedicated domain-specific language, and for that reason Parity offers the [ink!](#ink)
+   language.
+
+2. **Deposit**. Contracts must hold a deposit (named _ContractDeposit_ ) suitably large enough in
+   order to justify their existence on-chain. The developer needs to deposit this into the new contract
+   on top of the _ExistentialDeposit_.
+
+3. **Caching**. Contracts are cached by default and therefore means they only need to be deployed
+   once and afterward be instantiated as many times as you want. This helps to keep the storage load
+   on the chain down to the minimum. On top of this, when a contract is no longer being used and the
+   _existential deposit_ is drained, the code will be erased from storage (known as reaping).
+
+### Ink
+
+[ink!](https://github.com/use-ink/ink){target=\_blank} is a domain specific language for writing smart contracts
+in Rust and compiles to Wasm code.
+
+Here is the list of current resources available to developers who want to get started writing smart
+contracts to deploy on parachains based on Substrate.
+
+- [ink!](https://use.ink/){target=\_blank} - Polkadot's "native" smart contract stack, based on WebAssembly
+- [OpenBrush](https://docs.openbrush.io/){target=\_blank}: an `ink!` library providing standard contracts based on [PSP](https://github.com/w3f/PSPs){target=\_blank} with useful contracts and macros for building
+- [ink!athon](https://inkathon.xyz/){target=\_blank}: Starter kit for full-stack dApps with ink! smart contracts and frontend
+
+ink! has laid much of the groundwork for a new smart contract stack that is based on a Wasm virtual
+machine and compatible with Substrate chains.
+
+## Smart Contract Environments
+
+Many smart contract platforms are building to become a parachain in the ecosystem. A community created and maintained list of different smart contract platforms can be found at [PolkaProjects](https://www.polkaproject.com/#/projects?cateID=1&tagID=6){target=\_blank}. Additionally, information about ink smart contracts can be accessed at [use.ink](https://use.ink/#where-can-i-deploy-ink-contracts){target=\_blank}.
+
+#### Moonbeam
+
+- ink!: **Unsupported**
+- EVM (Solidity): [**Supported**](https://moonbeam.network/networks/moonbeam/){target=\_blank}
+
+[Moonbeam](https://moonbeam.network/){target=\_blank} is another project that is planning to deploy to Polkadot as a
+parachain and will support Ethereum compatible smart contracts. Since Moonbeam uses
+[Frontier](https://github.com/paritytech/frontier){target=\_blank}, an interoperability layer with existing Ethereum
+tooling, it will support all applications that are written to target the EVM environment with little
+friction.
```

Comment:
This should be written in the present tense since Moonbeam is already a parachain and already supports apps written to target EVM environments